browser-test/src/admin_api_keys.test.ts

```diff
@@ -1,13 +1,31 @@
+import { Browser, Page } from 'playwright'
 import {
   startSession,
   loginAsAdmin,
   AdminApiKeys,
   AdminPrograms,
+  closeWarningMessage,
+  endSession
 } from './support'
 
 describe('Managing API keys', () => {
+  let browser : Browser
+  let page : Page
+
+  beforeEach(async () => {
+    var session = await startSession()
+
+    browser = session.browser
+    page = session.page
+
+    await closeWarningMessage(page)
```

Comment:
suggestion (non-blocking): could call `closeWarningMessage` at the end of `startSession` so it happens for every test

Reply:
I feel like there was a reason I didn't do that when I first look at this a couple weeks back, but it's easy to test.

Reply:
@bion Tests still passed after moving it so it's in the startSession now